spyderlib/utils/icon_manager.py

```diff
@@ -40,7 +40,7 @@
     'arrow-continue':          [('spyder.continue',), {'color': '#3775a9'}],
     'arrow-step-in':           [('spyder.step-into',), {'color': '#3775a9'}],
     'arrow-step-out':          [('spyder.step-out',), {'color': '#3775a9'}],
-    'stop_debug':              [('spyder.stop',), {'color': '#3775a9'}],
+    'stop_debug':              [('fa.stop',), {'color': '#3775a9'}],
```

Comment:
When I first made the spyder font, I recreated a lot of icons duplicating some that were included in font-awesome.
I eventually removed most of them.

Here I stop using spyder.stop to the benefit of fa.stop and I removed 'stop' from the spyder font.